charts/nri-bundle-legacy/README.md

```diff
@@ -0,0 +1,178 @@
+# nri-bundle
```

Comment:
Should we update https://github.com/newrelic/helm-charts/blob/71c1a5c10c11be6ffb2d94c7bde00e5707dbc7a4/.gitignore#L2 as well?

Reply:
the gitignore basically

Reply:
I removed completely those lines as they are included in the line below:
```gitgitnore
# Downloaded chart dependencies
**/charts/*.tgz
```

Reply:
Solved in [#4f2332e](https://github.com/newrelic/helm-charts/pull/795/commits/4f2332e6fc57c5e9842ddb8e958681644a537814)